src/components/EditableCard.tsx

```diff
@@ -62,26 +67,40 @@ class EditableCard extends Component<IEditableCardProps> {
   }
 
   private get deleteButton() {
-    const { isGuarded, classNameSuffix, onDelete, isLoading, title } = this.props,
-      className = getBtnClassName('delete', classNameSuffix, title);
+    const {
+        classNameSuffix,
+        disabledDeleteTooltip,
+        disableDelete,
+        isGuarded,
+        isLoading,
+        model,
+        onDelete,
+        title,
+      } = this.props,
+      className = getBtnClassName('delete', classNameSuffix, title),
+      isDeleteDisabled = disableDelete && disableDelete(model);
 
     if (!onDelete) {
       return null;
     }
 
     return (
-      <GuardedButton
-        className={className}
-        confirm={true}
-        disabled={isLoading || this.isDeleting.isTrue}
-        icon={<DeleteOutlined />}
-        isGuarded={isGuarded}
-        onClick={this.handleDelete}
-        size="small"
-        type="danger"
-      >
-        Delete
-      </GuardedButton>
+      <Tooltip title={isDeleteDisabled ? disabledDeleteTooltip : ''}>
```

Comment:
if the button is disabled but the prop passed an empty string for `disabledDeleteTooltip`, does the tool tip render with an empty string or does not render at all?

Reply:
No tooltip is rendered when the string is empty!